v63004/outref/gtm3146.txt

```diff
@@ -0,0 +1,20 @@
+# Testing 
+# Create a 1 region DB region DEFAULT
+
+# Create a file, shell.csh, to be sourced by child shells : 
+
+cat shell.csh
+#!/usr/local/bin/tcsh -f
+set echo ; set verbose
+alias cp "jakewashere"
```

Comment:
Add a comment here that the MUPIP BACKUP done later (which internally does a "cp") used to fail in prior versions due to the "cp" alias here.

Reply:
The comment has been added to the generated shell.csh file. This way the comment will appear right below the cp alias when shell.csh is cat-ed to the screen.